src/components/button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import s from './Button.module.css';
+
+interface IButton {
+    buttonName: string;
```

Comment:
```suggestion
    name: string;
```
у тебя контекст button, не нужно мусорить в нейминг ещё раз писав button слово